observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -189,6 +189,46 @@ include::snippets/technology-preview.adoc[]
 <8> Defines how long metrics are exposed without updates. The default is `5m`.
 <9> Adds the metrics types and units suffixes. Must be disabled if the monitor tab in Jaeger console is enabled. The default is `true`.
 
+[id="prometheus-remote-write-exporter_{context}"]
+== Prometheus Remote Write Exporter
+
+The Prometheus Remote Write Exporter exports metrics to compatible back ends.
+
+[WARNING]
+====
+This exporter drops non-cumulative monotonic, histogram, and summary OTLP metrics.
+====
+
+:FeatureName: The Prometheus Remote Write Exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with an enabled Prometheus Remote Write Exporter
+[source,yaml]
+----
+# ...
+  config: |
```

Comment:
I think the pipe character might be throwing something off with the colour highlighting?

![Screenshot from 2024-08-30 12-42-48](https://github.com/user-attachments/assets/1b109fac-f328-40c0-91b9-e88fa37a01e3)

Reply:
It's not just the pipe character. This blue highlighting also appears for other YAML constructs.

Reply:
Maybe the extra hash here is throwing something off?

![Screenshot from 2024-08-30 17-17-55](https://github.com/user-attachments/assets/d93c994c-d99f-4f25-8bc2-d749f1f74ed1)

Reply:
I think Darragh is right that the `|` causes the stuff that follows to be all orange instead of a mix of colors (have seen this before). However, if the `|` is supposed to be there, this is an issue with however the syntax highlighting works rather than the content. Would be nice to figure that out, but well out of scope for the PR :)